agr_java_core/src/main/java/org/alliancegenome/core/config/ConfigHelper.java

```diff
@@ -67,7 +67,7 @@ public static void init() {
 
 		defaults.put(EXTRACTOR_OUTPUTDIR, "data");
 
-		defaults.put(NEO4J_HOST, "localhost");
+		defaults.put(NEO4J_HOST, "build.alliancegenome.org");
```

Comment:
Are you sure you want to change this?

Reply:
All good comments. Put back the changes that were meant to be local ones only.